rosdep/python.yaml

```diff
@@ -620,6 +622,7 @@ python-defusedxml:
   debian: [python-defusedxml]
   fedora: [python-defusedxml]
   gentoo: [dev-python/defusedxml]
+  opensuse: [python3-defusedxml]
```

Comment:
Thanks for review.  Compared with other distros, selected python2 as the default.